lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/head.html

```diff
@@ -1 +1,18 @@
 {% extends 'ace_common/edx_ace/common/base_head.html' %}
+{% block additional_styles %}
+<style type="text/css">
+    @media only screen and (max-width: 456px) {
+        .goals-banner-div {
+            background-image: url('{{image_url}}goalreminder-mobile-hero.png') !important;
+            height: 200px !important;
```

Comment:
Does this need to be in both px and rem?

Reply:
good question, i'll try out locally if this css trick still works with rem - will merge this for now though so that the emails I get before UX review have these changes already available